packages/common/base/src/blockchain/types/evm.ts

```diff
@@ -15,10 +15,17 @@ export type EVMBlockchain = ObjectValues<typeof EVMBlockchain>;
 export const EVM_CHAINS = objectValues(EVMBlockchain);
 
 export const EVMBlockchainTestnet = {
-    GOERLI: "goerli",
+    ARBITRUM_SEPOLIA: "arbitrum-sepolia",
+    BASE_GOERLI: "base-goerli",
+    BASE_SEPOLIA: "base-sepolia",
+    BSC_TESTNET: "bsc-testnet",
+    ETHEREUM_GOERLI: "ethereum-goerli",
     ETHEREUM_SEPOLIA: "ethereum-sepolia",
-    MUMBAI: "mumbai",
-    ZKATANA: "zkatana",
```

Comment:
did zkatana get dropped?

Reply:
sadly it's here, as a mainnet:
```
export const EVMBlockchain = {
    ETHEREUM: "ethereum",
    POLYGON: "polygon",
    BSC: "bsc",
    OPTIMISM: "optimism",
    ARBITRUM: "arbitrum",
    BASE: "base",
    ZORA: "zora",
    ARBITRUMNOVA: "arbitrumnova",
    ZKATANA: "zkatana",
} as const;
```
I raised the concern here: https://crossmint.slack.com/archives/C06LNUL04TH/p1709145570085719
But that's how it is right now in crossbit-main. This work is just to sync main and sdk. Any changes we would like to do would be have to be in a follow up.